MaterialDesignThemes.Wpf/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="XamlDisplayerPackage" version="1.0.0" targetFramework="net45" />
```

Comment:
@punker76 The reference is here

Reply:
@wongjiahau you must remove this and add this to the paket dependency and references file

Reply:
@punker76 Do you mind telling me where is the file??

Reply:
@wongjiahau https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/master/paket.dependencies
and
https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/paket.references

Reply:
@punker76 Thank you very much!

Reply:
@wongjiahau just forgot to say you must, after adding this to the files, call once '.paket\paket install' at the commandline and then check in the changed paket.lock file

Reply:
@punker76 I tried just now, but I'm really confused now, do you mind helping me to get the references done?

Reply:
No rush; I'm heading out to work so will pull in later.

Reply:
@ButchersBoy Thanks for your review and acceptance. 😄

Reply:
Thank you for contribution...I'm sure it's going to help a lot of people!

Reply:
@ButchersBoy  Before you merge I have to let you know that the part for Transitions is still incompleted yet, that part shall requires a great change so that each type of transitions can be demonstrated.

Reply:
Ok, well let's not worry about that for now, just leave that screen as is.